Formula/vile.rb

```diff
@@ -0,0 +1,35 @@
+class Vile < Formula
+  desc "Vi Like Emacs Editor"
+  homepage "https://invisible-island.net/vile/"
+  url "https://invisible-island.net/archives/vile/current/vile-9.8v.tgz"
```

Comment:
Is `v` part of the version? `brew` probably won't pick that up; see the output of `brew info vile` to check. If it doesn't:
```suggestion
  url "https://invisible-island.net/archives/vile/current/vile-9.8v.tgz"
  version "9.8v"
```

Reply:
```sh
brew info vile
vile: stable 9.8v
Vi Like Emacs Editor
https://invisible-island.net/vile/
/usr/local/Cellar/vile/9.8v (221 files, 7.8MB) *
```

It seem OK as is :)